R/pkg/R/mllib_classification.R

```diff
@@ -239,21 +253,57 @@ function(object, path, overwrite = FALSE) {
 setMethod("spark.logit", signature(data = "SparkDataFrame", formula = "formula"),
           function(data, formula, regParam = 0.0, elasticNetParam = 0.0, maxIter = 100,
                    tol = 1E-6, family = "auto", standardization = TRUE,
-                   thresholds = 0.5, weightCol = NULL, aggregationDepth = 2) {
+                   thresholds = 0.5, weightCol = NULL, aggregationDepth = 2,
+                   lowerBoundsOnCoefficients = NULL, upperBoundsOnCoefficients = NULL,
+                   lowerBoundsOnIntercepts = NULL, upperBoundsOnIntercepts = NULL) {
             formula <- paste(deparse(formula), collapse = "")
+            lrow <- 0
+            lcol <- 0
+            urow <- 0
+            ucol <- 0
```

Comment:
Here we can reduce from four parameters to two: ```nrow``` and ```ncol```, since the matrix format of ```lowerBoundsOnCoefficients``` and ```upperBoundsOnCoefficients``` should be consistent. If the input doesn't confirm with this rule, we need to throw exception.